numbers/src/test/java/org/dicio/numbers/lang/fr/DateTimeTest.java

```diff
@@ -0,0 +1,53 @@
+package org.dicio.numbers.lang.fr;
+
+import org.dicio.numbers.formatter.NumberFormatter;
+import org.dicio.numbers.test.DateTimeTestBase;
+import org.junit.Test;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static org.junit.Assert.assertEquals;
+
+public class DateTimeTest extends DateTimeTestBase {
+
+    @Override
+    public String configFolder() {
+        return "config/fr-fr";
+    }
+
+    @Override
+    public NumberFormatter buildNumberFormatter() {
+        return new FrenchFormatter();
+    }
+
+    @Test
+    public void testNiceDate() {
+        // just check that the NumberParserFormatter functions do their job
+        assertEquals("mercredi vingt-huit avril deux mille un",
+                pf.niceDate(LocalDate.of(2021, 4, 28)).get());
+        assertEquals("dimanche treize août",
+                pf.niceDate(LocalDate.of(-84, 8, 13)).now(LocalDate.of(-84, 8, 23)).get());
+    }
+
+    /*  Please note that there is two way of saying years and centuries before 2000. For exemple:
+        1. mille (thousand) neuf (nine) cent (hundred) quatre-vingt (90) quatre (4)
+        2. dix-neuf (nineteen) cent (hundred) quatre-vingt (90) quatre (4). (Slightly old-fashioned but common for years before 1900)
+    */
```

Comment:
It's perfectly ok for date time formatters to just return one possibility :-)

Reply:
Ok, we'll go with the first one then since as @MXC48 mentioned it is valid both in speech as well as in writing. :)